modules/ocf_mirrors/manifests/projects/archlinuxcn.pp

```diff
@@ -0,0 +1,33 @@
+class ocf_mirrors::projects::archlinuxcn {
+  file {
+    '/opt/mirrors/project/archlinuxcn':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/archlinuxcn/',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  file {
+    '/opt/mirrors/project/archlinuxcn/sync_password':
+        content   => lookup('mirrors::archlinuxcn_sync_password'),
+        show_diff => false,
+        owner     => mirrors,
+        group     => mirrors,
+        mode      => '0400';
+  }
+
+  ocf_mirrors::monitoring { 'archlinuxcn':
+    type          => 'unix_timestamp',
+    upstream_host => 'mirrors.tuna.tsinghua.edu.cn',
```

Comment:
I don't think this will work-- you probably have to set the `upstream_path` here to `archlinuxcn` (similar to [here](https://github.com/ocf/puppet/blob/master/modules/ocf_mirrors/manifests/projects/manjaro.pp))

Reply:
Fixed.